ipykernel/kernelspec.py

```diff
@@ -66,7 +66,7 @@ def get_kernel_dict(extra_arguments: list[str] | None = None) -> dict[str, Any]:
         "argv": make_ipkernel_cmd(extra_arguments=extra_arguments),
         "display_name": "Python %i (ipykernel)" % sys.version_info[0],
         "language": "python",
-        "metadata": {"debugger": _is_debugpy_available},
```

Comment:
I think `_is_debugpy_available` import/definition should have been removed as well?

Reply:
(I will include cleanup in a PR)

Reply:
Yes, thanks.